server/controllers/payroll/weekendConfig/index.js

```diff
@@ -52,10 +52,22 @@ function detail(req, res, next) {
 function create(req, res, next) {
   const sql = `INSERT INTO weekend_config SET ?`;
   const data = req.body;
+  const configuration = data.daysChecked;
+  let insertId;
+
+  delete data.daysChecked;
 
   db.exec(sql, [data])
```

Comment:
Go ahead and make this function `async/await` so you don't have to use `let`.  Like this:
```js
const { insertId } = await db.exec(sql, [data]);
const dataConfigured = configuration.map(id => ([id, insertId]));
await db.exec('INSERT INTO .... VALUES ?', dataConfigured]);
res.status(201).jsoin({ id : insertId });
```